H2TauTau/scripts/harvest.py

```diff
@@ -205,9 +205,6 @@ def get_options():
     parser.add_option("-s", "--subdir-pattern", dest="subdir_pattern",
                       default='*',
                       help='subdir pattern')
-    parser.add_option("-S", "--skim", dest="skim", action='store_true',
```

Comment:
yes, skimming feature is already implemented at Heppy level. Plus hardcoding skimming here can lead to confusion since the skimming is very likely to change and adapt regularly (as is right now).